src/javascript/app/pages/user/new_account/virtual_acc_opening.js

```diff
@@ -28,11 +30,14 @@ const VirtualAccOpening = (() => {
 
     const init = () => {
         $(form).setVisibility(1);
+        const mobileDevices = ['iPhone', 'iPad', 'Android'];
+        const dateFirstContact = toISOFormat(moment());
```

Comment:
as per Ashkan, this should be the first date that client visited our site. I suggest we store that date locally on page load, (and remove it after virtual account signup, and perhaps remove it on logging in as well) and use that value here.

Reply:
Alright, Will do. 

What is the default for it? (today?). If the user opened his/her activation link on another device, should we use the 'current date'?

Reply:
yes